ocaml/xapi/xapi_db_upgrade.ml

```diff
@@ -303,7 +303,9 @@ let upgrade_bios_strings =
         let oem_manufacturer =
           let test_path =
             Option.map
-              (fun _ -> "/tmp/previousInventory")
+              (fun _ ->
```

Comment:
Alternatively, we can just ditch this upgrade rule, which is for upgrades from XS 5.5, released on 2009.

Reply:
I still would prefer to have this as we might have to backport this to yangtze